packages/connectivity/src/scp-cf/destination/destination-accessor.ts

```diff
@@ -25,21 +23,18 @@ import type { DestinationAccessorOptions } from './destination-accessor-types';
  * @returns A promise resolving to the requested destination on success.
  */
 export async function useOrFetchDestination(
-  destination: Destination | DestinationNameAndJwt,
-  options: DestinationOptions = {}
+  destination: Destination | DestinationFetchOptions
 ): Promise<Destination | null> {
-  return isDestinationNameAndJwt(destination)
+  return isDestinationFetchOptions(destination)
     ? getDestination(
         destination.destinationName,
-        destination.jwt ? { userJwt: destination.jwt, ...options } : options
+        destination.jwt
```

Comment:
I would also adjust the API of the `getDestination()` method. This makes only sense if you have  `DestinationFetchOptions` in the other case we just sanatize the existing destination:
- create (use the existing) typeguard for `isDestination()` which decides if something is a destination or DestinationFetchOptions
- If it is the optins use then as argument for the getDestination

Reply:
Okay, I've changed the API of `getDestination` to use `DestinationFetchOptions`, along with all the affected tests